kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-job-template.yaml

```diff
@@ -67,6 +71,10 @@ spec:
               value: "%DOMAIN_HOME%"
             - name: DOMAIN_LOGS_DIR
               value: "%DOMAIN_ROOT_DIR%/logs/%DOMAIN_UID%"
+            - name: ISTIO_ENABLED
+              value: "%ISTIO_ENABLED%"
+            - name: ISTIO_READINESS_PORT
+              value: "%ISTIO_READINESS_PORT%"
```

Comment:
We shouldn't be requiring users to set these values - the Operator itself should set them for both the introspector job and the wl pods when istio is enabled via the domain yaml. If we want to keep it here, maybe rename it ISTIO_ENABLED_EXPERIMENTAL or at least put a clear comment above the env var stating that it's experimental.